src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -810,13 +810,18 @@ const WorkflowView = _.flow(
           div({ style: { display: 'flex', alignItems: 'baseline', minWidth: 'max-content' } }, [
             // This span is to prevent vertical resizing when the memory retry multiplier input is visible.
             span({ style: { marginTop: '0.5rem', marginBottom: '0.5rem' } }, [
-              span([
+              span({ style: { ...styles.checkBoxSpanMargins, marginLeft: 0 } }, [
                 h(LabeledCheckbox, {
                   disabled: currentSnapRedacted || !!Utils.computeWorkspaceError(ws),
                   checked: useCallCache,
                   onChange: v => this.setState({ useCallCache: v })
                 }, [' Use call caching'])
               ]),
+              h(InfoBox, [
+                'Call caching allows Terra\'s execution engine to detect when a job has been run in the past so that it doesn\'t have to re-compute results. ',
```

Comment:
Consistency suggestion - all of the other tool tips apply to "Terra's execution engine" but none of them mention it.
```suggestion
                'Call caching detects when a job has been run in the past so that it doesn\'t have to re-compute results. ',
```